examples/manual_reply/test/test.ts

```diff
@@ -69,18 +65,110 @@ const tests: Test[] = [
     {
         name: 'manual_update when calling ic.reply',
         test: async () => {
-            // TODO: Once ic.reply is implemented this shouldn't throw
-            try {
-                await manual_reply_canister.manual_update('accept');
+            const result = await manual_reply_canister.manual_update('accept');
 
-                return {
-                    ok: false
-                };
-            } catch (error) {
-                return {
-                    ok: (error as Error).message.includes('Reject code: 5')
-                };
-            }
+            return {
+                ok: result === 'accept'
+            };
+        }
+    },
+    {
+        name: 'reply with blob',
+        test: async () => {
+            const result = await manual_reply_canister.reply_blob();
+            const expectedResult = [83, 117, 114, 112, 114, 105, 115, 101, 33];
+
+            return {
+                ok:
+                    Array.isArray(result) &&
+                    result.every((byte, i) => byte === expectedResult[i])
+            };
+        }
+    },
+    {
+        name: 'reply with float32',
+        test: async () => {
+            const result = await manual_reply_canister.reply_float32();
+
+            return {
+                ok: result === 1245.677978515625 // TODO: This should not be changing
```

Comment:
What's wrong here?

Reply:
The canister returns `1245.678` in the typescript code but when we get outside the canister we have `1245.677978515625` instead. It shouldn't lose it's precision like that.